input/data/2014/2014ApJ...794L...1A/tev-000089-sed.ecsv

```diff
@@ -13,7 +13,6 @@
 # - source_id: 89
 # - source_name: TeV J1641-463
 # - reference_id: 2014ApJ...794L...1A
-# - url: https://www.mpi-hd.mpg.de/hfm/HESS/pages/publications/auxiliary/auxinfo_HESSJ1641
```

Comment:
Why did you remove that line?
It's good to have the URL where the data comes from, no?

In cases where I can't point to a URL where I got the data via email, I'm usually adding a comment line:
```
Data from <name> obtained <how> <when>
```
If you can and think it's useful, please add that info where you got the data from in a comment somewhere.